src/lang/Messages.properties

```diff
@@ -1491,6 +1491,9 @@ multiple.options.panel.ordered.move.down.button.tooltip = Moves the selected ele
 multiple.options.panel.ordered.move.bottom.button.label = Bottom
 multiple.options.panel.ordered.move.bottom.button.tooltip = Moves the selected element to bottom position.
 
+network.sslconnector.handshake.error.help = The following document may be of assistance in resolving this failure:\n{0} 
+network.sslconnector.handshake.error.help.url = https://github.com/zaproxy/zaproxy/wiki/FAQsslHandshake
```

Comment:
Worth changing to generic keys (if it's going to be used in the manual request dialogue and proxy thread)?